releases/catalog-prometheus-rule.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     prometheus: k8s
     role: prometheus-rulefiles
-    release: stakater-monitoring-prometheus-operator
+    release: stakater-monitoring-po
```

Comment:
This should be `stakater-monitoring-prometheus-operator`, i don't it should be renamed

Reply:
currently in our monitoring stack Prometheus operator has been named like this and this needs to match that release name. Secondly this was named as `po` intentionally to make the name a little short.